webxrlayers-1.bs

```diff
@@ -427,6 +427,29 @@ When <var ignore>setting the space on an cube layer</var> with {{XRSpace}} |spac
 
 </div>
 
+Spaces {#spaces}
+======
+
+{{XREquirectLayer}} and {{XRWebGLLayer}} don't have associated an {{XRSpace}} because they render to the full frame.
```

Comment:
did you mean XRProjectionLayer?

```suggestion
{{XRProjectionLayer}} and {{XRWebGLLayer}} don't have associated an {{XRSpace}} because they render to the full frame.
```